core/endpoint.go

```diff
@@ -364,6 +364,12 @@ func (endpoint *Endpoint) call(result *Result) {
 	}
 }
 
+func (endpoint *Endpoint) CloseHTTPConnection() {
+	if endpoint.Type() == EndpointTypeHTTP {
+		client.GetHTTPClient(endpoint.ClientConfig).CloseIdleConnections()
+	}
+}
```

Comment:
Should this perhaps be renamed to `Close()` just in case there are other endpoints types that need something to be closed as well in the future?

Also, perhaps a comment above that function would be useful -- I'd include a reference to the issue: `https://github.com/TwiN/gatus/issues/536`

Reply:
No problem!
I'll modify it quickly and commit it.
I'll ask for your review by then, thank you!